src/applications/avs/components/YourHealthInformation.jsx

```diff
@@ -260,6 +384,29 @@ const YourHealthInformation = props => {
         renderItem={renderAllergy}
         showSeparators
       />
+      <ItemsBlock
+        heading="My medications"
+        intro={medsIntro(avs)}
+        itemType="my-medications"
+        items={getMyMedications(avs)}
+        renderItem={renderMedication}
+        showSeparators
+      />
+      <ItemsBlock
+        heading="My VA supplies"
+        itemType="my-va-supplies"
+        items={getMySupplies(avs)}
+        renderItem={renderMedication}
+        showSeparators
+      />
+      <ItemsBlock
+        heading="Medications you are not taking"
+        intro="You have stated that you are no longer taking the following medications. Please remember to discuss each of these medications with your providers."
+        itemType="medications-not-taking"
+        items={getMyMedicationsNotTaking(avs)}
+        renderItem={renderNotTakingMedication}
+        showSeparators
+      />
```

Comment:
Based on the design screenshots, it seems there should there be a separator (such as a `<hr/>` or a border) between "Medications you are not taking" and other sections.

Current output:

![Screenshot from 2023-12-12 11-03-16](https://github.com/department-of-veterans-affairs/vets-website/assets/1299034/dfad85f5-7bcf-49d8-a928-866656f0881d)

Reply:
@radavis thanks for the thorough review! This comes from an edge case in the design - if a list of items only has one item, there are no HRs added. I just did a test with multiple supplies to confirm the correct behavior with multiple:

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/101649/61e09afc-ad4a-4f2b-83ea-a3e7ee2ec423)

Reply:
Ok, as long as you are aware.

Reply:
yup, it's intended behavior. cc @kristenmcconnell @ninaranusavice just in case I got it wrong 🙂